transformers/integration_tests/dog_deny_test.go

```diff
@@ -0,0 +1,4 @@
+package integration_tests
+
+//TODO: add integration test when mainnet has data
```

Comment:
Could you add a pending test here that will make some noise in the build, but won't fail? Not a huge deal, but better to have a reminder that runs than a comment.

Reply:
Sure... I think you can add the framework for it and us `XIt` to mark it pending. As long as it compiles.